step-templates/terraform-custom-script.json

```diff
@@ -0,0 +1,109 @@
+{
+    "Id": "1d2a62d0-1f12-4417-8566-4ca1b8e6a69c",
+    "Name": "Execute Custom Terraform Script with Package",
+    "Description": "Run a custom terraform script using a package and an Azure Account.\n\nE.g. run \"terraform taint some-resource\" in the context of your terraform package files and authenticated against Azure using credentials in an Octopus Azure Account variable. This step operates very similarly to the built-in Terraform Apply step but allowing you the control to specift custom terraform commands. Very useful for runbooks and terraform import.",
+    "ActionType": "Octopus.Script",
+    "Version": 1,
+    "CommunityActionTemplateId": null,
+    "Packages": [
+      {
+        "Name": "InfrastructurePackage",
+        "Id": "582b5176-aa5d-4166-a5fa-9b760a3b2d7d",
+        "PackageId": null,
+        "FeedId": "feeds-builtin",
```

Comment:
You can set the FeedId to null. If its set to a non-variable value, it will cause a validation error when imported and copied into a non-default space. See https://github.com/OctopusDeploy/Issues/issues/6733